src/data-dictionary/events/NrAiIncidentExternal/state.md

```diff
@@ -0,0 +1,8 @@
+---
+name: state
+type: attribute
+events:
+  - NrAiIncidentExternal
+---
+
+Whether the event should trigger a new incident or resolve an existing one. Enum: `trigger`, `resolve`, and `acknowledge`. Required.
```

Comment:
I'm not sure what `enum` is so I'm going to flag it, but if it's something users would know ignore this.